### PR TITLE
Updated Ukrainian translation

### PR DESCRIPTION
**Pull request by Tarteroycc**

### CODE REVIEW

File: src/main/resources/assets/jade/lang/uk_ua.json [hunk 1]
```diff
@@ -1,192 +1,363 @@
...
+	"jade.input_signal": "Введення: %d",
 	"tooltip.jade.villager_profession": "Професія: %s",
-	"gui.jade.plugin_settings": "Параметри плаґіна",
-	"gui.jade.forced_plugin_config": "Цей параметр керується сервером",
-	"gui.jade.translated_by": "placeholder %s",
-	"gui.jade.translators": "Altegar, BurrConnie",
+	"gui.jade.by": "Зроблено з %s від Snownee",
+	"gui.jade.by.hovered": "Перегляньте всі мої роботи!",
+	"gui.jade.translated_by": "Українська локалізація від %s",
+	"gui.jade.translators": "Altegar, Tarteroyc",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/uk_ua.json]: BurrConnie should not be removed

**Review by Snownee — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/jade/lang/uk_ua.json (hunk 1 above)
Comment (Tarteroycc) [reply src/main/resources/assets/jade/lang/uk_ua.json]: Okay, I will update soon

**Comment by Snownee** [comment]
I've just added some entries, please translate them as well

**Comment by Tarteroycc** [comment]
> I've just added some entries, please translate them as well

Updated.

**Comment by Snownee** [comment]
please do not change the indent and resolve the conflicts

**Comment by Tarteroycc** [comment]
> please do not change the indent and resolve the conflicts

Perhaps I accidentally made a retreat, but I don't see where it is.

**Comment by Tarteroycc** [comment]
I have re-translated the English file so everything should be fine.

**Comment by Tarteroycc** [comment]
> I have re-translated the English file so everything should be fine.

If in the future there is a problem, write what is the essence and I will try to solve it

File: src/main/resources/assets/jade/lang/uk_ua.json [hunk 2]
```diff
@@ -1,191 +1,387 @@
...
+	"jade.theme.jade.create": "Створити",
+	"jade.collectingItems": "Збираю речі...",
+	"config.jade.plugin_jade.block_properties": "Властивості блоку",
+	"jade.block_explosion_resistance": "Вибухостійкість: %s",
+	"jade.block_destroy_time": "Час знищення: %s",
+	"jade.block_jump_factor": "Фактор стрибка: %s",
+	"jade.block_speed_factor": "Фактор швидкості: %s",
+	"jade.block_ignite_odds": "Шанс підпалу: %s",
+	"jade.block_burn_odds": "Шанси згоріти: %s",
+	"jade.ignore_list.comment": "Це список ігнорування для цілі Jade. Ви можете додати ідентифікатори реєстру до списку значень.",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/uk_ua.json]: The word "values" should not be translated

File: src/main/resources/assets/jade/lang/uk_ua.json [hunk 3]
```diff
@@ -1,191 +1,387 @@
...
+	"jade.access.creeper.powered": "Заряджений %s",
+	"jade.access.zombie_villager.curing": "Вилікування %s",
+	"jade.access.goat.hornless": "Безрогий %s",
+	"jade.access.wither.powered": "Посилений %s",
+	"jade.access.entity.sheared": "Стрижений %s",
+	"jade.access.entity.saddled": "Осідланий %s",
+	"jade.access.entity.baby": "Дитинча %s",
+	"jade.access.entity.angry": "Злий %s",
+	"jade.access.bee.nectar": "Нектар %s",
+	"jade.access.slime.size": "{1,choice,1#Small|2≤Medium|4≤Large|8≤Huge} {0}",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/uk_ua.json]: please translate Small, Medium, Large and Huge

**Review by Snownee — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/jade/lang/uk_ua.json (hunk 3 above)
Comment (Tarteroycc) [reply src/main/resources/assets/jade/lang/uk_ua.json]: It's done

**Comment by Snownee** [comment]
Thank you!

**Pull request merged**